doc/source/dev/api-dev/array_api.rst

```diff
@@ -57,7 +57,8 @@ values:
 
 Note that the above example works for PyTorch CPU tensors. For GPU tensors or
 CuPy arrays, the expected result for ``vq`` is a ``TypeError``, because ``vq``
-uses compiled code in its implementation, which won't work on GPU.
+uses compiled code in its implementation, which won't work on GPU, and there
+are currently no GPU specific implementations to dispatch to.
```

Comment:
```suggestion
are currently no GPU specific implementations to delegate to.
```
I was under the impression this was the preferred phrasing

Reply:
Indeed. I must have missed this one after the first round of reviews.